clmm/theory/parent_class.py

```diff
@@ -334,15 +334,25 @@ def eval_reduced_tangential_shear(self, r_proj, z_cl, z_src, z_src_model='single
                     (Applegate et al. 2014; https://arxiv.org/abs/1208.0605) to evaluate tangential reduced shear;
                 * `schrabback18`: use the equation (12) in Cluster Mass Calibration at High Redshift \
                     (Schrabback et al. 2017; https://arxiv.org/abs/1611.03866) to evaluate tangential reduced shear;
-                
+        
+        pdz: one parameter function
+            The probability density distribution of the redshift. This function takes /
+            the redshift as its only parameter and returns its probability. It is /
```

Comment:
To me, a PDF should not return a probability if we are not taking the integral of it. But I am also not sure how to best describe this part.